assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -373,6 +374,7 @@ private void configureMultiUserMode(
     install(new OrganizationJpaModule());
 
     install(new KeycloakModule());
+    install(new KeycloakUserRemoverModule()); // TODO remove before merge to the master
```

Comment:
Please address that TODO

Reply:
_remove before merge to the master_

Reply:
Why not remove it now? PR can't be ready to review if it has TODOs, can it?